<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="easydoc.qrc">
    <normaloff>:/images/active.png</normaloff>:/images/active.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_files">
     <property name="title">
      <string>Open &amp;Recent</string>
     </property>
     <addaction name="actionNothing_yet"/>
    </widget>
    <widget class="QMenu" name="menu_Import">
     <property name="title">
      <string>&amp;Import...</string>
     </property>
     <addaction name="actionGameLua"/>
    </widget>
    <widget class="QMenu" name="menu_Export">
     <property name="title">
      <string>&amp;Export...</string>
     </property>
     <addaction name="actionFunctionList"/>
     <addaction name="actionWiki"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="action_open"/>
    <addaction name="menuRecent_files"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="menu_Export"/>
    <addaction name="menu_Import"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSave_Wiki"/>
    <addaction name="actionRemember_last_file"/>
    <addaction name="actionLarge_Font"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuOptions"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset resource="easydoc.qrc">
     <normaloff>:/images/document-open.png</normaloff>:/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="easydoc.qrc">
     <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="easydoc.qrc">
     <normaloff>:/images/document-new.png</normaloff>:/images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNothing_yet">
   <property name="text">
    <string>nothing yet</string>
   </property>
  </action>
  <action name="actionGameLua">
   <property name="text">
    <string>GameLua.h</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="easydoc.qrc">
     <normaloff>:/images/active.png</normaloff>:/images/active.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="icon">
    <iconset resource="easydoc.qrc">
     <normaloff>:/images/File.png</normaloff>:/images/File.png</iconset>
   </property>
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionFunctionList">
   <property name="text">
    <string>FunctionList</string>
   </property>
  </action>
  <action name="actionPlain_Text">
   <property name="text">
    <string>Plain Text</string>
   </property>
  </action>
  <action name="actionWiki">
   <property name="icon">
    <iconset resource="easydoc.qrc">
     <normaloff>:/images/matt-icons_emblem-recycle.png</normaloff>:/images/matt-icons_emblem-recycle.png</iconset>
   </property>
   <property name="text">
    <string>Wiki</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave_HTML">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save HTML</string>
   </property>
  </action>
  <action name="actionSave_Wiki">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save Wiki</string>
   </property>
  </action>
  <action name="actionRemember_last_file">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remember last file</string>
   </property>
  </action>
  <action name="actionLarge_Font">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Large Font</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="easydoc.qrc"/>
 </resources>
 <connections/>
</ui>
